application/pages/WorldListPage.cpp

```diff
@@ -0,0 +1,94 @@
+//
+// Created by robotbrain on 8/18/15.
+//
+
+#include "WorldListPage.h"
+#include "ui_WorldListPage.h"
+#include "minecraft/WorldList.h"
+#include "dialogs/ModEditDialogCommon.h"
+#include <QEvent>
+#include <QKeyEvent>
+
+WorldListPage::WorldListPage(BaseInstance *inst, std::shared_ptr<WorldList> worlds, QString id,
+							 QString iconName, QString displayName, QString helpPage,
+							 QWidget *parent)
+	: QWidget(parent), ui(new Ui::WorldListPage)
+{
+	ui->setupUi(this);
+	ui->tabWidget->tabBar()->hide();
+	m_inst = inst;
```

Comment:
Might want to put all of these into the initialization list of the constructor instead.